build.xml

```diff
@@ -373,8 +554,29 @@
       retrieveTarget="${source} sprint ${version}" />
   </target>
 
+  <!-- SYNTAX: ant backup-dev -Dtype=package|destructiveChanges -Dversion=3.6 change for story S-502094 starts-->
+  <target name="backup-dev" depends="check-files-exist" if="${type}.exists">

```

Comment:
All of the added backup logic looks great for cmcdev, cmcqa, and full - thank you!